<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1709</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>751</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>751</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabLexico">
     <attribute name="title">
      <string>Lexico</string>
     </attribute>
     <widget class="QTextEdit" name="textLexico">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSintactico">
     <attribute name="title">
      <string>Sintactico</string>
     </attribute>
     <widget class="QTextEdit" name="textSintactico">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSemantico">
     <attribute name="title">
      <string>Sem√°ntico</string>
     </attribute>
     <widget class="QTextEdit" name="textSemantico">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHashTable">
     <attribute name="title">
      <string>Hash Table</string>
     </attribute>
     <widget class="QTextEdit" name="textHashTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInterCode">
     <attribute name="title">
      <string>CodigoIntermedio</string>
     </attribute>
     <widget class="QTextEdit" name="textInterCode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabErroresLex">
     <attribute name="title">
      <string>Errores</string>
     </attribute>
     <widget class="QTextEdit" name="textErroresLex">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1491</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1709</width>
     <height>26</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuBuild_and_Debug">
    <property name="title">
     <string>Build and Debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAppearance">
    <property name="title">
     <string>Appearance</string>
    </property>
    <addaction name="actionSet_Dark_Mode"/>
    <addaction name="actionSet_Light_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionIncrease_Font_Size"/>
    <addaction name="actionDecrease_Font_Size"/>
   </widget>
   <widget class="QMenu" name="menuLexico">
    <property name="title">
     <string>Lexico</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSintactico">
    <property name="title">
     <string>Sintactico</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSemantico">
    <property name="title">
     <string>Semantico</string>
    </property>
   </widget>
   <widget class="QMenu" name="iconNew">
    <property name="toolTip">
     <string>New File</string>
    </property>
    <property name="title">
     <string>Compilar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>new.png</normaloff>new.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="iconOpen">
    <property name="title">
     <string>lol</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>open.png</normaloff>open.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="iconClose">
    <property name="title">
     <string>Close</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>close.png</normaloff>close.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="iconSave">
    <property name="title">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>save.png</normaloff>save.png</iconset>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild_and_Debug"/>
   <addaction name="menuAppearance"/>
   <addaction name="menuLexico"/>
   <addaction name="menuSintactico"/>
   <addaction name="menuSemantico"/>
   <addaction name="iconNew"/>
   <addaction name="iconOpen"/>
   <addaction name="iconClose"/>
   <addaction name="iconSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionChange_Font_Size">
   <property name="text">
    <string>Change Font Size</string>
   </property>
  </action>
  <action name="actionSet_Dark_Mode">
   <property name="text">
    <string>Set Dark Mode</string>
   </property>
  </action>
  <action name="actionSet_Light_Mode">
   <property name="text">
    <string>Set Light Mode</string>
   </property>
  </action>
  <action name="actionIncrease_Font_Size">
   <property name="text">
    <string>Increase Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionDecrease_Font_Size">
   <property name="text">
    <string>Decrease Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+_</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normalon>close.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
